src/sentry/features/manager.py

```diff
@@ -282,3 +286,7 @@ def get_feature_objects(self) -> Mapping["Project", Feature]:
 
         cls = self._manager._get_feature_class(self.feature_name)
         return {obj: cls(self.feature_name, obj) for obj in self.objects}
+
+    @property
+    def main_entity(self) -> Organization | User:
+        return self.organization or self.actor
```

Comment:
Will this still be correct for project features?

Reply:
@markstory It must work because we are just using `batch.organization` for this check